flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/sink/KafkaWriterStateSerializerTest.java

```diff
@@ -29,7 +30,8 @@
  * Tests for serializing and deserialzing {@link KafkaWriterState} with {@link
  * KafkaWriterStateSerializer}.
  */
-public class KafkaWriterStateSerializerTest extends TestLogger {
+@ExtendWith(TestLoggerExtension.class)
```

Comment:
Using the `META-INF/services/org.junit.jupiter.api.extension.Extension` is the better approach here.